src/main/java/com/amadeus/Booking.java

```diff
@@ -0,0 +1,23 @@
+package com.amadeus;
+
+import com.amadeus.booking.FlightOrder;
+
+public class Booking {
+  private Amadeus client;
+
+  /**
+   * <p>
+   *   A namespaced client for the
+   *   <code>/v1/booking/flightOrder</code> endpoints.
+   * </p>
+   */
+  public FlightOrder flightOrder;
+
+  public Booking(Amadeus client) {
+    this.client = client;
+  }
+
+  public FlightOrder flightOrder(String flightOrderId) {
```

Comment:
Can this function name be more descriptive, like "getFlightOrder" or "generateFlightOrder"?

Reply:
If I change the name of the function then the API would be called as `amadeus.booking.getFlightOrder("eJzTd9f3NjIJdzUGAAp%2fAiY=").get();` instead of `amadeus.booking.flightOrder("eJzTd9f3NjIJdzUGAAp%2fAiY=").get();` that we specified.